### PR TITLE
Dir cleanup

### PR DESCRIPTION
**Pull request by mfridman**

### CODE REVIEW

File: cmd/screenshot/main.go [hunk 1]
```diff
@@ -4,13 +4,13 @@ import (
...
 
-	"github.com/pressly/screenshot/internal/screenshotserver"
+	"github.com/pressly/screenshot/internal/server"
 	"github.com/pressly/screenshot/rpc/screenshot"
 )
 
 func main() {
 
-	srv := &screenshotserver.Server{}
+	srv := &server.Server{}
```
Comment (mfridman) [inline cmd/screenshot/main.go:13]: Reads a bit funny, I guess that's why they suggest 

> Do not name the package something generic like api, client or service; name it after your service. Remember that the package is going to be imported by other projects that will likely import other clients from other services as well.

cc @VojtechVitek

File: cmd/screenshot/main.go [hunk 2]
```diff
@@ -4,13 +4,13 @@ import (
 	"log"
 	"net/http"
 
-	"github.com/pressly/screenshot/internal/screenshotserver"
```
Comment (VojtechVitek) [inline cmd/screenshot/main.go:7]: can we drop "internal" too?

File: cmd/screenshot/main.go (hunk 1 above)
Comment (VojtechVitek) [reply cmd/screenshot/main.go:13]: yeah, ideally it would read `screenshot.Server{}`

Comment (VojtechVitek) [reply cmd/screenshot/main.go:13]: but whatever .. server won't be used by the project users directly

File: cmd/screenshot/main.go (hunk 2 above)
Comment (mfridman) [reply cmd/screenshot/main.go:7]: ye

File: server/server.go [hunk 3]
```diff
@@ -17,6 +17,10 @@ func (s *Server) Image(ctx context.Context, req *pb.RequestImage) (*pb.Resp, err
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
+
+	chrome, headless.NewImage
```
Comment (VojtechVitek) [inline server/server.go]: typo?

Comment (mfridman) [reply server/server.go]: fixed.

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Pull request merged**